_posts/2025-05-19-announcement.md

```diff
@@ -1,8 +1,249 @@
 ---
 layout: post
-title: "Introducing Marin"
-author: Marin Team
+title: "Introducing Marin: An Open Lab"
+authors:
+- David Hall
+- Ahmed Ahmed, Chris Chou, Abhinav Garg, Rohith Kuditipudi, Will Held, Nikil Ravi, Herumb Shandilya, Jason Wang
+- Jason Bolton, Siddharth Karamcheti, Suhas Kotha, Tony Lee, Nelson Liu, Joel Niklaus, Ashwin Ramaswami, Kamyar Salahi, Kaiyue Wen, Chi Heem Wong, Sherry Yang, Ivan Zhou
+- Percy Liang
 date: 2025-05-19
 categories: blog
 ---
-Hello there!
+
+Open-source *software* is a success story:
+It powers the world's digital infrastructure.
+It allows anyone in the world to contribute based on merit.
+It leads to greater innovation, collaboration, and security.
+
+Open-source AI is not there yet.
+We have open-weight models (e.g., Llama, DeepSeek, Gemma), sometimes mistakenly called open-source models,
+but the code and data (recipe) used to produce the model are not released.
+There has been a movement over the last few years to build [open-source](https://opensource.org/ai/open-source-ai-definition) models.
+[Eleuther AI](https://www.eleuther.ai/),
+[Allen Institute for AI (AI2)](https://allenai.org/),
+[Hugging Face](https://huggingface.co/),
+[BigScience](https://bigscience.huggingface.co/),
+[BigCode](https://www.bigcode-project.org/),
+[DataComp-LM](https://www.datacomp.ai/dclm/),
+[MAP-Neo](https://map-neo.github.io/),
+[LLM360](https://www.llm360.ai/),
+[Together AI](https://www.together.ai/),
+[NVIDIA](https://www.nvidia.com/),
+and others have taken the path (so far) less traveled.
+These teams not only have released model weights,
+but also the code and data recipe used to produce the model.
+These assets have enabled others (including Marin!) to build their own models
+and push forward innovation.
+
+![Marin]({{ site.baseurl }}/assets/images/posts/announcement-open.png)
+
+We would like to go one step further.  In open-source software, it is easy for
+anyone to download the code, extend it, and contribute back to the community.
+This is because over the last two decades, the community has developed mature
+tools: versioned code hosting (e.g., GitHub), continuous integration, and
+governance strategies.  Such infrastructure doesn't exist yet for the development of
+foundation models.  You can't train locally, and you can't just log into someone
+else's GPU or TPU cluster.
+
+## A new form of openness
+
+Marin is an **open lab**, in which the research and development of models
+is completely transparent from day 1 (that's today).  We leverage GitHub to organize the open lab's
+efforts, piggybacking off of tools and processes developed for open-source software.
+Here's how it works:
+
+1. Each **experiment** (whether it be on the wishlist, currently running, or
+completed) is tracked by a **GitHub issue**---see an
+[example](https://github.com/marin-community/marin/issues/1183).  These GitHub
```

Comment:
I think putting an easier example than MoE might be better for the first-time reader ?

Reply:
I think it's okay - it's a representative example